plat/socionext/uniphier/include/platform_def.h

```diff
@@ -27,26 +27,28 @@
 #define PLAT_MAX_OFF_STATE		2
 #define PLAT_MAX_RET_STATE		1
 
-#define UNIPHIER_SEC_DRAM_BASE		0x80000000
+#define BL2_BASE			0x80000000
```

Comment:
@masahir0y 
Check failures were static analysis issues due to:
misra_violation: [Required] MISRA C-2012 Rule 7.2 violation: Unsigned constants must be declared with U or u suffix.

Could you add ULL-suffix to:
- BL2_XXX
- UNIPHIER_SEC_DRAM_XXX
- BL31_XXX
- BL32_XXX
- UNIPHIER_ROM_REGION_XXX
- UNIPHIER_OCM_REGION_XXX
- UNIPHIER_REG_REGION_XXX

Everything else looked fine.

Thanks!